lessons/Projects/project-threats/langs/en-us/pages/threats-project-notes-2.adoc

```diff
@@ -0,0 +1,28 @@
+[.landscape]
+= Analysis of Research Plan Predicted Outcomes (2)
+
+Complete the table below with your partner.
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^5,^.^5", options="header", stripes="none"]
```

Comment:
It really seems like the first column should be even smaller. Also, all this empty space feels like we're begging for kids to write as messily as possible.  Can we either fill these cells with blank lines, or replace the table with numbered-questions followed by blank lines?

(Ditto for threats-project-notes-2.adoc)